packages/core/src/InstanceSettings.ts

```diff
@@ -133,6 +134,22 @@ export class InstanceSettings {
 		return this.settings.tunnelSubdomain;
 	}
 
+	/**
+	 * Whether this instance is running inside a Docker container.
+	 *
+	 * Based on: https://github.com/sindresorhus/is-docker
+	 */
+	@Lazy
```

Comment:
Actually I started thinking is `Lazy` a good name here? The getter does get executed immediately it is used, so I don't think it's really "lazy" per se? Would it make more sense to name this `Memoized`?

Reply:
I don't have a string opinion either way. I named it lazy because of my previous experiences in kotlin. 
If we can get a consensus on this from @despairblue and @ivov, then I'd gladly rename this to whatever everyone agrees on.

Reply:
To me lazy means something is not evaluated until it is accessed, which getters are by nature. E.g.

```ts
class Example {
  // Evaluated when initialized
  notLazy = expensiveComputation()

  // Evaluated only when accessed
  get lazy() {
    return expensiveComputation()
  }
}
```

Reply:
Agree, deferring computation and caching its result are two separate things. Suggestion: `@MemoizedGetter`

Reply:
Do we need the `Getter` suffix? Maybe we can have that info in the JSDoc, and just called it `@Memoized`? or do you think we might want to support memoization on class methods as well at some point?

Reply:
I like the explicitness of the `-Getter` suffix, but happy already with `@Memoized`